config.tfvars

```diff
@@ -37,4 +37,15 @@ db_instance_class    = "db.t3.micro"
 db_allocated_storage = 1000
 db_iops              = 1000
 
-
+# Bamboo license
+# To avoid storing license in a plain text file, we recommend storing it in an environment variable prefixed with `TF_VAR_` and keep the below line commented out
+# If deploy in a non-prod environment and plain-text value does not expose a security risk, feel free to uncomment the following line and supply the license here
```

Comment:
I would skip the `non-prod` environment message as we don't recommend this for prod environments.